### DISCUSSION TITLE
How to get ForumsTopics Ids to delete using a Bot

### DISCUSSION THREAD

**Discussion by richbars** [python-telegram-bot/python-telegram-bot#4223] [Q&A] [answered]
Hi, Community!

I want to get the ids of the ForumTopics that are in the "file_names" list and create a function that deletes them from my chat, is there any way to do that?

```python
async def deleteForumTopic(update: Update, context: CallbackContext):
    file_names = fileNames()
    bot = context.bot
    chatId = "-1002064646692"
```

**Comment by Poolitzer** [comment]
~~[Hi, not with the bot API/PTB respectively~~

Sorry misread check first comment

**Reply by Bibo-Joshi** [reply]
Well, there is the [`delete_forum_topic`](https://docs.python-telegram-bot.org/en/stable/telegram.bot.html#telegram.Bot.delete_forum_topic) method. So deleting forum topics is surely possible. Reading the ids from the file is rather basic python though and not part of this issue tracker.

**Reply by Poolitzer** [reply]
Oh wow I totally misunderstood, I thought they want to get The forum topic ids

**Reply by richbars** [reply]
I understand there is delete_forum_topic, but is there any easy way to collect all ForumsTopic Ids to delete them all at once?

**Reply by Bibo-Joshi** [reply, marked as answer]
You'll have to keep track of them. There is no API method to get all currently existing topic IDs
